data/networks.json

```diff
@@ -57,6 +57,10 @@
         "url": "wss://polygon-testnet.public.blastapi.io",
         "type": "evm-ws"
       },
+      {
```

Comment:
This seems to be like ZetaChain's mainnet gRPC endpoint in the Polygon Mumbai Testnet section.